docker/Dockerfile

```diff
@@ -1,12 +1,16 @@
 ARG BASE_IMAGE=ubuntu:22.04
 FROM "$BASE_IMAGE" AS base
+ARG TARGETARCH
+ARG TARGETARCH="${TARGETARCH:-amd64}"
 
 FROM base AS bootstrap
 
 SHELL ["/bin/bash", "-c"]
 
 ENV LANG=C.UTF-8
 
+RUN echo "Build for $TARGETARCH"
```

Comment:
Please don't, each RUN creates a new layer

Reply:
Then how do I print logs?

Reply:
You shouldn't print this in prod.
For debugging it doesn't matter.
If you have to print something, do it within a useful later.